<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>成品入库单</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_add">
           <property name="text">
            <string>增加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_edit">
           <property name="text">
            <string>编辑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_del">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_save">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cancel">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_check">
           <property name="text">
            <string>审核</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_print">
           <property name="text">
            <string>打印</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_excel">
           <property name="text">
            <string>Excel导入</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>198</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>入库单号</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>入库日期</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="7">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>入库部门</string>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>入库仓库</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="in_num"/>
        </item>
        <item row="2" column="8">
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item row="2" column="5">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="3" column="6">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="10">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>批设价格</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="3" column="2" colspan="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>118</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>供应商</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="3" column="7">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>单据类型</string>
          </property>
         </widget>
        </item>
        <item row="3" column="8">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="3" column="9">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="btn_copy">
          <property name="text">
           <string>复制</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="main_tab"/>
      </item>
      <item>
       <widget class="QTableView" name="sec_tab">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>总件数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>总重量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>总金重</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>总成本</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_10"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionadd">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actionrow">
   <property name="text">
    <string>删除选中行</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
